### DISCUSSION TITLE
float_seq without discretization?

### DISCUSSION THREAD

**Discussion by ElkanaBaris** [RUCAIBox/RecBole#1737] [Q&A]
Hi,

I'm trying to add an input of float_seq, some external embedding, to my model.
It seems that although I didn't configure it, the embedding undergo a discretization process.
I don't want to embed it again.
Is that true? 
What is the thought process behind it?
How can I disable this functionality?

Thanks!

BTW - you have an awesome repo!

P.S. attached is the code from recbole/data/dataset/dataset.py:
<img width="773" alt="image" src="https://user-images.githubusercontent.com/104759389/232438470-4fa6e29e-2907-43be-b3df-79f5dd2968f2.png">

**Comment by kerengaiger** [comment]
I also struggle with that issue
